libs/s25main/figures/nofWellguy.h

```diff
@@ -32,7 +32,7 @@ class nofWellguy : public nofWorkman
     void DrawWorking(DrawPoint drawPt) override;
     /// Gibt die ID in JOBS.BOB zurück, wenn der Beruf Waren rausträgt (bzw rein)
     // TODO:der Brunnentyphat keine ID in JOBS.BOB
```

Comment:
do you want to translate the comment to english? (and perhaps add a name for the `100` (`JOBS_TO_CARRIER_ID_OFFSET` ?)

Reply:
Done